README.md

```diff
@@ -52,6 +52,16 @@ app.get('/status', ensureLoggedIn, require('express-status-monitor')())
 
 Credits to [@mattiaerre](https://github.com/mattiaerre)
 
+Example using [http-auth](https://www.npmjs.com/package/http-auth)
+```javascript
+const auth = require('http-auth');
+const basic = auth.basic({realm: 'Monitor Area'}, function(user, pass, callback) {
+  callback(user === 'username' && pass === 'password');
+});
+
+app.get('/status', auth.connect(basic), require('express-status-monitor')());
+```
```

Comment:
I would also add a "Credits to @cristianossd" section as above; and if we would like to be more consistent we need to change the link pointing to **connect-ensure-login** in the section above